rest/routes/lastVisited.go

```diff
@@ -28,19 +25,18 @@ func StoreLastVisitedPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = service.HandlePostLastVisitedPages(userId, currentPage)
+	err = service.HandlePostLastVisitedPages(recentPages.Pages, userId)
 	if err != nil {
 		panic(err)
 	}
-
 	pages, err := service.GetUsersLastVisitedPages(userId)
 
 	if err != nil {
 		panic(err)
```

Comment:
Note for the future, we should standardize response handling including errors. I've started a little bit on the dashboard PR.